content/hi/container.md

```diff
@@ -0,0 +1,27 @@
+---
+शीर्षक: कंटेनर
+स्थिति: Completed
+श्रेणी: टेक्नोलॉजी
+---
+
+### यह क्या है
+--- 
+    
```

Comment:
```suggestion
```

Reply:
We don't need horizontal rule. Please check [style guide](https://glossary.cncf.io/style-guide/) and English version of pages to get an idea of formatting